qa/common/perf_analyzer_input_data_json/repeat_int32_data.json

```diff
@@ -0,0 +1,31 @@
+{
+    "data": [
+        {
+            "IN": {
+                "content": [
+                    4,
+                    2,
+                    0,
+                    1
+                ],
+                "shape": [
+                    4
+                ]
+            },
+            "DELAY": {
+                "content": [
+                    1,
+                    2,
+                    3,
+                    4
+                ],
+                "shape": [
+                    5
```

Comment:
shape mismatch between IN and DELAY. 
Either change shape to 4 or add an entry to Delay.content and IN.content and update shape for IN

Reply:
Typo. Changed to `4` 👍